### PR TITLE
Revert "Introducing a facade API : (#53)"

### PR DESCRIPTION
**Pull request by ehsavoie**
This reverts commit b3b6794544ffd56b42b98ce3b7530ac55c361b8c.

### CODE REVIEW

File: examples/liberty-car-booking/pom.xml [hunk 1]
```diff
@@ -1,120 +1,175 @@
...
-                <role>PROJECT LEAD</role>
-            </roles>
-        </developer>
-    </developers>
+	<developers>
+		<developer>
+			<name>Buhake Sindi</name>
+			<timezone>+2</timezone>
+			<roles>
+				<role>PROJECT LEAD</role>
```
Comment (Emily-Jiang) [inline examples/liberty-car-booking/pom.xml:18]: In MicroProfile, there is no `Project Lead` for the specifications.

Comment (Emily-Jiang) [reply examples/liberty-car-booking/pom.xml:18]: Please remove the section of `<roles>`

**Review by Emily-Jiang — COMMENTED** [review]
Thank you @ehsavoie for the PR! It looks good to me except one minor comment.

**Comment by ehsavoie** [comment]
@Emily-Jiang fixed

**Review by Emily-Jiang — APPROVED** [review]
LGTM

**Review by TheEliteGentleman — APPROVED** [review]
I'm approving this so that we can refactor/test this accordingly.

**Pull request merged**